subworkflows/local/bam_variant_calling_somatic_controlfreec/main.nf

```diff
@@ -27,16 +27,39 @@ workflow BAM_VARIANT_CALLING_SOMATIC_CONTROLFREEC {
 
     FREEC_SOMATIC(controlfreec_input, fasta, fasta_fai, [], dbsnp, dbsnp_tbi, chr_files, mappability, intervals_bed, [])
 
-    ASSESS_SIGNIFICANCE(FREEC_SOMATIC.out.CNV.join(FREEC_SOMATIC.out.ratio, failOnDuplicate: true, failOnMismatch: true))
+    //Filter the files that come out of freec somatic as ASSESS_SIGNIFICANCE only takes one cnv and one ratio file
+    //Creates empty channel if file is missing
+    cnv_files = FREEC_SOMATIC.out.CNV
+    .map{ meta, cnv ->
+        def meta_clone_tumor = meta.clone()
```

Comment:
```suggestion
```

Reply:
You are right, this was leftover code from the previous version of the script, updated it.